bluemix/configuration/core_config/iam_token.go

```diff
@@ -38,6 +40,15 @@ type AccountsInfo struct {
 	Valid        bool   `json:"valid"`
 }
 
+type Authn struct {
```

Comment:
This field contains the relevant profile and compute resource information in an iam token